### DISCUSSION TITLE
How to implement Server-Sent-Events SSE?

### DISCUSSION THREAD

**Discussion by Roberto99Roberto99** [adonisjs/core#3664] [Help]
**How to implement Server-Sent-Events SSE?**

I need a one-way connection to notify my client. In the AdonisJS documentation I only found this using WS, but I think it's a bit exaggerated to use WebSocket as a solution.

I don't know how to implement SSE in AdonisJS. I found only one package from version 4, but I couldn't implement it in version 5.

**Comment by maximehuylebroeck** [comment]
Hi,
As a temporary workaround you can always use polling. 
I'm also interested in SSE though.
